### PR TITLE
add nuxt-swaggerjsonapi

### LINKED ISSUE #92: Add nuxt-swaggerjsonapi
Module repo:
https://github.com/516310460/SwarggerJSON-API

### PR DESCRIPTION
**Pull request by 516310460**
Resolves #92

### CODE REVIEW

File: modules/nuxt-swaggerjsonapi.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: nuxt-swaggerjsonapi
+description: 'Efficient development, fast copy and paste use for Nuxt.js'
+repo: 516310460/nuxt-swaggerjsonapi
+npm: nuxt-swaggerjsonapi
+icon: nuxt-swaggerjsonapi.svg
```
Comment (Geminii) [inline modules/nuxt-swaggerjsonapi.yml:5]: You define your icon but not added into the project ?

File: modules/nuxt-swaggerjsonapi.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+name: nuxt-swaggerjsonapi
+description: 'Efficient development, fast copy and paste use for Nuxt.js'
+repo: 516310460/nuxt-swaggerjsonapi
```
Comment (Geminii) [inline modules/nuxt-swaggerjsonapi.yml:3]: repo is : 516310460/SwarggerJSON-API on Github

File: modules/nuxt-swaggerjsonapi.yml (hunk 1 above)
Comment (516310460) [reply modules/nuxt-swaggerjsonapi.yml]: I left the icon empty

Comment (516310460) [reply modules/nuxt-swaggerjsonapi.yml]: I added my own icon

File: modules/nuxt-swaggerjsonapi.yml (hunk 2 above)
Comment (516310460) [reply modules/nuxt-swaggerjsonapi.yml:3]: Change the repurchase address to https://github.com/516310460/nuxt-swaggerjsonapi

**Review by 516310460 — COMMENTED** [review]
Reload

**Review by 516310460 — COMMENTED** [review]
Resolved!

**Comment by pi0** [comment]
Hi @516310460 and thanks for PR. I've added a pending label to this PR until fixing some issues by https://github.com/516310460/nuxt-swaggerjsonapi/issues/1

**Pull request closed without merging**